pipelines/visitQAEllipPlots.yaml

```diff
@@ -0,0 +1,367 @@
+description: |
+    Make visit level ellipticity plots for QA, makes plots of the e1, e2 and size measurements
+    for the PSF shape measurements and the shape measurements in the shape slot, typically
+    CModel. Makes 18 plots that get saved as pngs.
+tasks:
+  plot_e1_scatter_visit:
```

Comment:
You have wonderful little summaries for all the individual plots in `visitQAPlots.yaml`, and now these ones are maybe feeling sad and left out.